app/views/changeContactDetails/IndividualPhoneView.scala.html

```diff
@@ -40,6 +40,7 @@
                 label = LabelViewModel(messages(s"individualPhone.heading")).asPageHeading()
             ).withHint(HintViewModel(HtmlContent(messages("individualPhone.hint"))))
             .withWidth(Fixed20)
+            .withAttribute("tel" -> "tel")
```

Comment:
The ticket states to put attributes `autocomplete to email, tel or name` and then `type = tel or email`. How this `withAttribute("tel" -> "tel")` achieves that?